test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -952,6 +1003,18 @@ public EndpointState getEndpointStateForEndpoint(InetAddressAndPort ep)
             {
                 return endpoints.get(ep);
             }
+
+            @Override
+            public void notifyFailureDetector(Map<InetAddressAndPort, EndpointState> remoteEpStateMap)
+            {
+
+            }
+
+            @Override
+            public void applyStateLocally(Map<InetAddressAndPort, EndpointState> epStateMap)
+            {
```

Comment:
if we were testing paxos this would be wrong... I made gossip work with this simulation logic (see https://issues.apache.org/jira/browse/CASSANDRA-18917) so we could always leverage that if we cared... but we would need to figure out a bug found In that JIRA (a node Is added to gossip but a small subset never learns this)...

So for this patch, I went with no-op

Reply:
Please add this info as a comment in the codebase